src/components/Dropdown/index.js

```diff
@@ -0,0 +1,29 @@
+// @flow
+import * as React from 'react'
```

Comment:
I prefer `import React, { type Node } from 'react'` for clarity, but maybe that’s just me